### ISSUE TITLE
How should wa update behave?

### ISSUE THREAD

**Issue by bucaran** [wa/wahoo#59]

**Comment by s0r00t** [comment]
Linked to #38 ?

**Comment by bucaran** [comment]
@gretel @s0r00t 

+ `wa update ` should update all packages. I'll add that soon.
+ `wa update` should ... ? (@s0r00t's PR)

**Comment by bucaran** [comment]
@s0r00t What about simply updating Wahoo without ever touching any remotes at all. Whether is origin or upstream, Wahoo should always update against `fish-shell/wahoo`.

**Comment by gretel** [comment]
@bucaran i'd really like that low-bound approach. but i just had some pseudo-code done in this form here, seems to work, guess just the value of $uri would have to be set at the right place.
```
set -l uri 'git@github.com:fish-shell/wahoo.git'
set -l ref (git config --get remote.upstream.url)
if test -z $ref
  git remote add upstream $uri
else if test $ref -neq $uri
  git remote set-url upstream $uri
end
```

**Comment by bucaran** [comment]
@gretel LGTM

**Comment by gretel** [comment]
i'm sorta busy and wouldn't be able to code and PR right now.. tomorrow or so..

**Comment by bucaran** [comment]
:+1:

**Comment by s0r00t** [comment]
That's what my PR does... in upgraded. 
@bucaran : That's what my PR does. Check if the fork got an upstream (the original repo) remote, then update against it. But if it's a clone of the repo without fork, update origin (which is also the original repo).

**Comment by bucaran** [comment]
`wa update` should update the framework as well as plugins/themes.

**Comment by bucaran** [comment]
@s0r00t Sorry for the long delay. Landed in 1.0.0.

It will try to update Wahoo. Then it will update the current theme and installed packages. :smile:
